python/opae.admin/opae/admin/tools/fpgasupdate.py

```diff
@@ -423,7 +423,9 @@ def update_fw(args, pac):
     retries = 0
     timeout = 1.0
     max_retries = 60 * 5
-    while status.value == 'read_file' or status.value == 'preparing':
+    # read_file is now deprecated. Leaving it in for backwards compat.
+    while status.value == 'read_file' or status.value == 'reading' \
+        or status.value == 'preparing':
```

Comment:
I like to avoid using '\' as much as possible.
Will it work if we change the while clause to something like this?
```Python
    while status.value in ['read_file', 'reading' , 'preparing']:
```

Reply:
Thanks, I like this better.  Fixed